resume-buddy/src/main/java/com/google/sps/servlets/RevieweeDataServlet.java

```diff
@@ -40,14 +40,12 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
     datastore.put(revieweeEntity);
 
-    /* TODO: add a response.sendRedirect() to wherever the user should
-     * be navigated to next (either homepage or resume review page)
-     */ 
+    response.sendRedirect("resume-review.html");
```

Comment:
Do the reviewer and reviewee get some kind of notification that their form was successfully submitted or do they just get directed back to `resume-review.html`?

Reply:
for now they just get redirected but that's a good idea I'll add that in another PR